### DISCUSSION TITLE
CO2 Emissions for CANDU nuclear (Canada)

### DISCUSSION THREAD

**Discussion by christopheradlam** [electricitymaps/electricitymaps-contrib#4791] [General]
Back in the late 90's the Canadian Nuclear Society performed an extensive lifecycle emissions analysis on Canadian nuclear (our CANDU fleet). 

The authors were:
S.    Andseta
M. J. Thompson
Sheridan College, Mississauga, Ontario
 
J.  P. Jarrell
Cameco Corporation, Saskatoon, Saskatchewan
 
D. R. Pendergast
Atomic Energy of Canada Limited, 2251 Speakman Drive
Mississauga, Ontario, L5K 1B2  

Website for the CNS is: https://www.cns-snc.ca/

The study is not located on their current website, however a copy of the document, including all references, can be found here:
http://www.computare.org/Support%20documents/Publications/Life%20Cycle.htm

Which shows actual lifecycle emissions for CANDU nuclear at 3.2gCO2/kWh. 

My hope is that we can use this study as the basis for the emissions intensity for the Canadian nuclear plants in Ontario.

**Comment by IkeBottema** [comment]
> Back in the late 90's the Canadian Nuclear Society performed an extensive lifecycle emissions analysis on Canadian nuclear (our CANDU fleet).
> 
> The authors were: S. Andseta M. J. Thompson Sheridan College, Mississauga, Ontario
> 
> J. P. Jarrell Cameco Corporation, Saskatoon, Saskatchewan
> 
> D. R. Pendergast Atomic Energy of Canada Limited, 2251 Speakman Drive Mississauga, Ontario, L5K 1B2
> 
> Website for the CNS is: https://www.cns-snc.ca/
> 
> The study is not located on their current website, however a copy of the document, including all references, can be found here: http://www.computare.org/Support%20documents/Publications/Life%20Cycle.htm
> 
> Which shows actual lifecycle emissions for CANDU nuclear at 3.2gCO2/kWh.
> 
> My hope is that we can use this study as the basis for the emissions intensity for the Canadian nuclear plants in Ontario.

Excellent point Chris.

**Comment by VIKTORVAV99** [comment]
CC @pierresegonne

**Comment by pierresegonne** [comment]
thanks for digging this information :)

Do you know by chance if there is a more recent source out there? I'm thinking something published in the 2010s at least?

**Reply by christopheradlam** [reply]
OK, I found a currently maintained lifecycle emissions database from the Canadian government:
https://www.canada.ca/en/environment-climate-change/services/managing-pollution/fuel-life-cycle-assessment-model.html

If you download and install the utility and import the latest database, Lifecycle emissions for CANDU nuclear show as 6.49gCO2/kWh. 

Not as low as the CNS figures, clearly, but about half what the IPCC shows.

**Reply by christopheradlam** [reply]
This paper is Ontario-specific and is from 2008:
https://cna.ca/wp-content/uploads/2014/05/Comparative-Life-Cycle-Analysis-of-Base-Load-Electricity-in-Ontario.pdf

It has emissions for non-hydro baseload generators in Ontario:
1. CANDU nuclear: 1.84gCO2/kWh
2. Coal (not applicable): 1,051.22gCO2/kWh
3. Gas: 540.39gCO2/kWh

**Reply by christopheradlam** [reply]
Another publication, which is oft cited by industry including Ontario's grid operator (IESO) and our main power generation company, OPG, Mallia and Lewis (2013) is also Ontario-specific. 
This publication has CANDU lifecycle emissions at 4.8gCO2/kWh
https://link.springer.com/article/10.1007/s11367-012-0501-0

If you have somebody in Poland or have a Polish VPN, it looks like you can view the whole text (it has emissions for all of Ontario's sources) here:
https://www.infona.pl/resource/bwmeta1.element.springer-f7c2058b-3e15-33d2-92d1-23da4ed7ee19

One thing about this article is that it takes into consideration the grade of uranium ore used for fuel fabrication in Canada, which is a significant factor in reduced emissions compared to other jurisdictions with respect to mining. 

Reference:
https://www.world-nuclear.org/information-library/nuclear-fuel-cycle/mining-of-uranium/uranium-mining-overview.aspx

_Most of the uranium ore deposits at present supporting these mines have average grades in excess of 0.10% of uranium – that is, greater than 1000 parts per million. In the first phase of uranium mining to the 1960s, this would have been seen as a respectable grade, but today **some Canadian mines have huge amounts of ore up to 20% U average grade**._

**Comment by pierresegonne** [comment]
Great! Thanks for the digging! I encourage you to then open a pull request that enacts that change. 

Let me know if you don't know how to do that or want some support :)

**Reply by VIKTORVAV99** [reply]
Should they implement this for gas and nuclear or just nuclear?

**Reply by christopheradlam** [reply]
> Should they implement this for gas and nuclear or just nuclear?

I think if we can get the Mallia and Lewis numbers we might as well do all sources for Ontario, as they should all be in that report.

**Reply by christopheradlam** [reply]
OK, yeah, going to need some guidance on that, have not opened a pull request before.

**Reply by MisterClean** [reply]
@christopheradlam I just submitted a pr (https://github.com/electricitymaps/electricitymaps-contrib/pull/4822) using the Mallia & Lewis figure for CANDU

**Reply by VIKTORVAV99** [reply]
> @christopheradlam I just submitted a pr (#4822) using the Mallia & Lewis figure for CANDU

If you can I would suggest adding @christopheradlam as a co-contributor to the git commit and as a contributor to the Ontario zone as well.

**Reply by christopheradlam** [reply]
Thank you very much!

**Reply by christopheradlam** [reply]
Hey, thanks for all your help guys! I see it has been committed, how long does it take for it to show up in the app?

**Reply by VIKTORVAV99** [reply]
There is no defined release schedule but I'd guess there will be a release early next week and as soon as a new release has been published [here](https://github.com/electricitymaps/electricitymaps-contrib/releases) it should be visible in the map (the next time the state files update and they update every hour on the dot) for the live data.
Since this also changes the historical co2 intensity that could potentially take a little longer but it will catch up eventually.

**Reply by christopheradlam** [reply]
Awesome, thank you for getting back to me so quickly, greatly appreciate it. I am going to purchase the Mallia and Lewis paper, review, and, if all is in order, create a separate submission to update other emissions factors for Ontario.

**Reply by VIKTORVAV99** [reply]
Looking forward to it! I don't know how much it costs but I would recommend trying to access it in a way that don't cost you money (library, university, etc) before purchasing it.
